R/synchronization.R

```diff
@@ -0,0 +1,24 @@
+#' Check whether all versions NEWS.md have been git-tagged
+#'
+#' @return `NULL`
+#' @export
+#'
+#' @example man/examples/check-tags.R
+check_tags <- function() {
+  versions <- get_news_headers()[["version"]]
+  tags <- gsub("^v", "", gert::git_tag_list()[["name"]])
+
+  untagged_versions <- versions[!(versions %in% tags)]
+  if (length(untagged_versions) == 0) {
+    return(NULL)
+  }
+
+  rlang::warn(
```

Comment:
Should that info be part of the upcoming fledgeling data structure?

Reply:
tracked there now, thanks